src/Symfony/Component/Form/Tests/Extension/Core/DataTransformer/BooleanToStringTransformerTest.php

```diff
@@ -60,11 +60,52 @@ public function testReverseTransformFailsIfInteger()
         $this->transformer->reverseTransform(1);
     }
 
+    /**
+     * @expectedException \Symfony\Component\Form\Exception\TransformationFailedException
+     */
+    public function testReverseTransformFailsIfUnexspectedValue()
+    {
+        $this->transformer->reverseTransform('foobar');
+    }
+
     public function testReverseTransform()
```

Comment:
jepp, good idea ... that change revealed a little bug which was not compliant to the documented behaviour from the `UPGRADE.md`:
https://github.com/symfony/symfony/pull/18005/files#diff-fbec32e9e5675480183eb6488f464fc6R89